src/docs/feature-flags.mdx

```diff
@@ -10,7 +10,7 @@ You can find a list of features available by looking at
 `sentry/features/__init__.py`. They're declared on the `FeatureManager` like so:
 
 ```python
-default_manager.add("organizations:onboarding", OrganizationFeature)  # NOQA
+default_manager.add("organizations:onboarding", OrganizationFeature, True)  # NOQA
 ```
```

Comment:
Can we add something here to explain what `True` is doing? Or link them to the explanation below? Unless the intent is for all feature flagging to be done through flagr, this might confuse some people when they read this.

Reply:
The true could also be a kwarg and implicitly explain what it is everywhere you see it.

Reply:
went for both
```python
# Don't set entity_feature, or set it to False if you don't plan to use Flagr
default_manager.add("organizations:onboarding", OrganizationFeature, entity_feature=True)
```